src/Raven.Studio/typescript/viewmodels/database/tasks/ongoingTasks.ts

```diff
@@ -73,6 +75,20 @@ class ongoingTasks extends viewModelBase {
     private initObservables() {
         this.myNodeTag(this.clusterManager.localNodeTag());
         this.serverWideBackupUrl = appUrl.forServerWideBackupList();
+        
+        this.taskNameToCount = ko.pureComputed<dictionary<number>>(() => {
+            var tasksNameToCount: { [id: string] : number; } = {};
+          
+            tasksNameToCount["External Replication"] = this.replicationTasks().length;
```

Comment:
and return this as object w/o assigning props line by line. I know you spent last week in java, but hopefully this is TS and we have nice syntax for doing this. ;-)

Reply:
so it goes like this:

return {
   "External replication" : this.replicationTasks().length,
....
}

Reply:
done - thanks